tests/test_sales.py

```diff
@@ -47,18 +48,19 @@ def test_create_order_apples_oranges():
         },
     )
     assert response.status_code == 200
-    id = response.json()["id"]  # unique id
+    id_ = response.json()["id"]  # unique id
     assert response.json() == {
-        "id": id,
+        "id": id_,
         "datestamp": "2011/12/02",
         "buyer": "ana",
```

Comment:
how do we know the buyer of this order?

Reply:
I assume this is already available in the test database table. See my last comment - we can discuss this!